src/app/routes/connector-ui/asset-page/asset-cards/asset-cards.component.html

```diff
@@ -12,9 +12,11 @@
     <mat-card-subtitle>{{ asset.creatorOrganizationName }}</mat-card-subtitle>
   </mat-card-header>
   <mat-card-content class="flex flex-col space-y-[16px]">
-    <!-- Description -->
-    <div *ngIf="asset.description" class="whitespace-pre-line truncate-lines-5">
-      {{ asset.description }}
+    <!-- Description Short Text -->
+    <div
+      class="whitespace-pre-line truncate-lines-5"
+      [class.italic]="!asset.descriptionShortText">
+      {{ asset.descriptionShortText ?? 'No description.' }}
     </div>
```

Comment:
third time, right? maybe it's time for a short component that does this, one that does not contain the div container because the component itself will leave behind a tag that can be styled by `@HostBinding("class.abc") cls = true;`